7.rb

```diff
@@ -0,0 +1,7 @@
+def find_fib(n)
+    if n <= 2
+        return 1
+    else
+        return find_fib(n-1) + find_fib(n-2)
+    end
```

Comment:
Use tenary operation for short if/else statement
Add space around `-` operation